web/src/admin/AdminInterface/SidebarEntries/GenericTypesController.ts

```diff
@@ -0,0 +1,55 @@
+import { ReactiveControllerHost } from "lit";
```

Comment:
This thing is both straightforward and pretty magical.  All our fetchers are simple.